OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/ContactSyncAdapterService.java

```diff
@@ -143,6 +143,13 @@ public IBinder onBind(Intent intent) {
     }
 
     public static void requestContactsSync() {
+        // if user has disabled automatic sync, do nothing
```

Comment:
Good point, I'll keep this in mind.. I'll make this change in my next pull request.